tmux-gruvbox-dark.conf

```
@@ -1,32 +1,26 @@
 ## COLORSCHEME: gruvbox dark
 set-option -g status "on"
 
-# default statusbar colors
-set-option -g status-bg colour237 #bg1
-set-option -g status-fg colour223 #fg1
+# default statusbar color
+set-option -g status-style bg=colour237,fg=colour223 # bg=bg1, fg=fg1
 
 # default window title colors
-set-window-option -g window-status-bg colour214 #yellow
-set-window-option -g window-status-fg colour237 #bg1
+set-window-option -g window-status-style bg=colour214,fg=colour237 # bg=yellow, fg=bg1
 
-set-window-option -g window-status-activity-bg colour237 #bg1
-set-window-option -g window-status-activity-fg colour248 #fg3
+set-window-option -g window-status-activity-style bg=colour237,fg=colour248 # bg=bg1, fg=fg3
 
 # active window title colors
-set-window-option -g window-status-current-bg default
-set-window-option -g window-status-current-fg colour237 #bg1
+set-window-option -g window-status-current-style bg=red,fg=colour237 # fg=bg1
 
 # pane border
-set-option -g pane-active-border-fg colour250 #fg2
-set-option -g pane-border-fg colour237 #bg1
+set-option -g pane-active-border-style fg=colour250 #fg2
+set-option -g pane-border-style fg=colour237 #bg1
 
 # message infos
-set-option -g message-bg colour239 #bg2
-set-option -g message-fg colour223 #fg1
+set-option -g message-style bg=colour239,fg=colour223 # bg=bg2, fg=fg1
 
 # writting commands inactive
-set-option -g message-command-bg colour239 #fg3
-set-option -g message-command-fg colour223 #bg1
+set-option -g message-command-style bg=colour239, fg=colour223 # bg=fg3, fg=bg1
```

```suggestion
set-option -g message-command-style bg=colour239,fg=colour223 # bg=fg3, fg=bg1
```

Seems there's excess whitespace.